analytics/article_impressions.js

```diff
@@ -117,15 +171,21 @@ if(location.pathname.match('/article/') || location.pathname.match('/articles'))
 
   var trackImpressions = function() {
     // Find signups, cta, share buttons, artist follow, toc, and image sets
-    var items = $('.articles-es-cta, .article-social, .article-section-image-set, .article-section-toc, .artist-follow, .article-section-callout');
-
+    var items = $('.articles-es-cta__form, .article-es-form, .article-social, .article-share-fixed-fullscreen, .article-section-image-set, .article-section-toc, .article-sa-sticky-header.visible, .article-sa-related, .artist-follow, .article-section-callout, .article-related-widget');
```

Comment:
Just a style thing, but this could be broken down into multiple lines. Maybe you could define the selector string prior with multilines, and then say `var items = $(selectorString)`